resources.go

```diff
@@ -150,30 +150,44 @@ func Provider() tfbridge.ProviderInfo {
 					Value:   "https://synthetics.newrelic.com/synthetics/api/v3",
 				},
 			},
+			"infrastructure_api_url": {
+				Default: &tfbridge.DefaultInfo{
+					EnvVars: []string{"NEWRELIC_INFRASTRUCTURE_API_URL"},
+				},
+			},
+			"nerdgraph_api_url": {
+				Default: &tfbridge.DefaultInfo{
+					EnvVars: []string{"NEWRELIC_NERDGRAPH_API_URL"},
+				},
+			},
 		},
 		PreConfigureCallback: preConfigureCallback,
 		Resources: map[string]*tfbridge.ResourceInfo{
-			"newrelic_alert_channel":              {Tok: makeResource(mainMod, "AlertChannel")},
-			"newrelic_alert_condition":            {Tok: makeResource(mainMod, "AlertCondition")},
-			"newrelic_alert_policy":               {Tok: makeResource(mainMod, "AlertPolicy")},
-			"newrelic_alert_policy_channel":       {Tok: makeResource(mainMod, "AlertPolicyChannel")},
-			"newrelic_dashboard":                  {Tok: makeResource(mainMod, "Dashboard")},
-			"newrelic_infra_alert_condition":      {Tok: makeResource(mainMod, "InfraAlertCondition")},
-			"newrelic_nrql_alert_condition":       {Tok: makeResource(mainMod, "NrqlAlertCondition")},
-			"newrelic_synthetics_alert_condition": {Tok: makeResource(syntheticsMod, "AlertCondition")},
-			"newrelic_synthetics_monitor":         {Tok: makeResource(syntheticsMod, "Monitor")},
-			"newrelic_synthetics_monitor_script":  {Tok: makeResource(syntheticsMod, "MonitorScript")},
-			"newrelic_insights_event":             {Tok: makeResource(insightsMod, "Event")},
-			"newrelic_plugins_alert_condition":    {Tok: makeResource(pluginsMod, "AlertCondition")},
+			"newrelic_alert_channel":                {Tok: makeResource(mainMod, "AlertChannel")},
+			"newrelic_alert_condition":              {Tok: makeResource(mainMod, "AlertCondition")},
+			"newrelic_alert_policy":                 {Tok: makeResource(mainMod, "AlertPolicy")},
+			"newrelic_alert_policy_channel":         {Tok: makeResource(mainMod, "AlertPolicyChannel")},
+			"newrelic_dashboard":                    {Tok: makeResource(mainMod, "Dashboard")},
+			"newrelic_infra_alert_condition":        {Tok: makeResource(mainMod, "InfraAlertCondition")},
+			"newrelic_nrql_alert_condition":         {Tok: makeResource(mainMod, "NrqlAlertCondition")},
+			"newrelic_synthetics_alert_condition":   {Tok: makeResource(syntheticsMod, "AlertCondition")},
+			"newrelic_synthetics_monitor":           {Tok: makeResource(syntheticsMod, "Monitor")},
+			"newrelic_synthetics_monitor_script":    {Tok: makeResource(syntheticsMod, "MonitorScript")},
+			"newrelic_insights_event":               {Tok: makeResource(insightsMod, "Event")},
+			"newrelic_plugins_alert_condition":      {Tok: makeResource(pluginsMod, "AlertCondition")},
+			"newrelic_application_label":            {Tok: makeResource(pluginsMod, "ApplicationLabel")},
+			"newrelic_synthetics_label":             {Tok: makeResource(pluginsMod, "SyntheticsLabel")},
+			"newrelic_synthetics_secure_credential": {Tok: makeResource(pluginsMod, "SyntheticsSecureCredential")},
```

Comment:
Should these two be in `syntheticsMod`? And maybe the names should omit `Synthetics` then?